plugins/polio/api/vaccines/stock_management.py

```diff
@@ -736,6 +747,12 @@ def update(self, instance, validated_data):
 class OutgoingStockMovementViewSet(VaccineStockSubitemBase):
     serializer_class = OutgoingStockMovementSerializer
     model_class = OutgoingStockMovement
+    permission_classes = [
+        lambda: VaccineStockManagementPermission(
+            non_admin_perm=permission.POLIO_VACCINE_STOCK_MANAGEMENT_READ,
+            admin_perm=permission.POLIO_VACCINE_STOCK_MANAGEMENT_WRITE,
```

Comment:
It's confusing that you put them in "non-admin, admin" order where in the constructor you did "admin, non-admin."

This should be consistent